app/code/Magento/Widget/Test/Mftf/Test/AdminContentWidgetsDisplayOnSpecificEntitiesTest.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminContentWidgetsDisplayOnSpecificEntitiesTest">
+        <annotations>
+            <features value="Widget"/>
+            <stories value="Widget parameter configuration"/>
+            <title value="Admin content widgets display on specific entities test"/>
+            <description value="Admin should be able to select specific entities for widgets"/>
+            <severity value="CRITICAL"/>
+            <group value="widget"/>
+        </annotations>
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="LoginAsAdmin"/>
+        </before>
+        <after>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+        <amOnPage url="{{AdminNewWidgetPage.url}}" stepKey="createWidgetPage"/>
+        <actionGroup ref="AdminCreateSpecificEntityWidgetActionGroup" stepKey="fillForm">
```

Comment:
Great job! I would suggest adjusting this test slightly. Ideally, if there's no chooser (original issue) we should have a failing test on an assertion. Wi the current test we will have rather an error (absent selector) than a failing assertion. 

My recommendation is to make the test work in the following way. 
- Select categories radio
- Select container value
- **Assert** that the chooser is present on the page

I hope it makes sense

Reply:
@rogyar Updated PR per your feedback